integrations/pjrt/python_packages/iree_rocm_plugin/setup.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3
+
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Early splice the _setup_support directory onto the python path.
+import os
+from pathlib import Path
+import sys
+
+THIS_DIR = os.path.realpath(os.path.dirname(__file__))
+sys.path.insert(0, os.path.join(THIS_DIR, "..", "_setup_support"))
+
+import iree_pjrt_setup
+from setuptools import setup, find_namespace_packages
+
+README = r"""
+OpenXLA PJRT Plugin for ROCM
+"""
+
+# Setup and get version information.
+CMAKE_BUILD_DIR_ABS = os.path.join(THIS_DIR, "build", "cmake")
+
+
+class CMakeBuildPy(iree_pjrt_setup.BaseCMakeBuildPy):
+    def build_default_configuration(self):
+        print("*****************************", file=sys.stderr)
+        print("* Building base runtime     *", file=sys.stderr)
+        print("*****************************", file=sys.stderr)
+        self.build_configuration(
+            os.path.join(THIS_DIR, "build", "cmake"),
+            extra_cmake_args=("-DIREE_EXTERNAL_HAL_DRIVERS=ROCM",),
```

Comment:
Thanks Stella for this cool addition! Overall rocm bit looks good to me!

Reply:
Go ahead and have a once over of all of it and approve if looks good. None of this is in CI yet, and we have a couple more iterations before usable generally...